### ISSUE TITLE
Documentation: Sunset procedures for SIGs/WGs

### ISSUE THREAD

**Issue by parispittman** [kubernetes/community#2029]
We should have documentation on the why and how to dissolve a SIG/WG, turn a WG into a SIG (or vice versa), combining efforts, etc. #2025 inspired this as we have a few more areas to fix/remove/modify.

The why should come from steering? (I could have glanced over the why in one of the new governance docs) The how could come from contribex. 

/sig contributor-experience
/steering

**Comment by jdumars** [comment]
https://github.com/kubernetes/community/pull/2629

**Comment by spiffxp** [comment]
I opened an an initial stab #2629 that adds a section, and notes we can retire sigs due to inactivity.  

Based on comments in https://groups.google.com/a/kubernetes.io/d/msg/steering/IRFRNwFjQYA/wYd5WUw8BQAJ and https://github.com/kubernetes/community/pull/2629#discussion_r215471944 it sounds like the steering committee would like to discuss more specifics before opening another PR with more detailed criteria.

**Comment by parispittman** [comment]
@spiffxp in the meantime, me/jorge/both can try to work up a draft of logistical items to add to that after the SC convo happens. I'll add a gdoc link here if we get something together before you do.

**Comment by parispittman** [comment]
me + @castrojo will work on this after election cycle.

**Comment by dims** [comment]
@parispittman #2629 has merged. Do we need to do more here?

**Comment by parispittman** [comment]
@dims - yes, we need to document the logistics of retirement. ie - process to follow to archive any repos, slack channels, mailing lists, etc.

**Comment by parispittman** [comment]
working document until we draft in GH
https://docs.google.com/document/d/1_venhAEU1ndOD57Jcz03E645xEtj4eAkrRIOaWjSsJ4/edit?usp=sharing

**Comment by castrojo** [comment]
I am confused as to why we have two `sig-governance.md` documents? I've linked to the one in the steering directory from the root one but imo it's confusing.

**Comment by michelleN** [comment]
Update: #2832 looks good is waiting on an official lgtm and unhold from @spiffxp before merge. Aaron added a follow up item #2857 as well.

**Comment by philips** [comment]
#2832 is merged.

**Comment by spiffxp** [comment]
/close
I'll close this and take on the followup item separately as it's out of scope for this issue
